wiki/Guides/OpenGL_Support_Issues/de.md

```diff
@@ -1,30 +1,30 @@
 
 ## Fehlender oder alter Treiber
 
-Wenn Sie nicht den richtigen Grafikkartentreiber installiert haben, wird Windows den "Microsoft Basic Display Adapter" als Ersatztreiber verwenden, welches für DirectX Spiele **funktioniert**, aber sehr langsam. Bei OpenGL wird es nicht funktionieren, daher sollten Sie auf Nummer sichergehen und den richtigen Treiber verwenden.
+Wenn du nicht den richtigen Grafikkartentreiber installiert hast, wird Windows den "Microsoft Basic Display Adapter" als Ersatztreiber verwenden, welcher für DirectX Spiele **funktioniert**, aber sehr langsam. Bei OpenGL wird es nicht funktionieren, daher solltest du auf Nummer sichergehen und den richtigen Treiber verwenden.
 
-Als erstes lassen Sie uns überprüfen, ob dies auf Sie zutrifft:
+Als erstes überprüfe, ob dies auf dich zutrifft:
 
-- Rechtsklick auf 'Dieser PC' und wählen Sie 'Eigenschaften' aus oder drücken Sie Windows-Taste + Pause auf Ihrer Tastatur.
-- Wählen Sie Geräte-Manager auf der linken Seite aus.
+- Rechtsklick auf 'Dieser PC' und wähle 'Eigenschaften' aus oder drücke `Windows-Taste` + `Pause` auf Ihrer Tastatur.
+- Wähle Geräte-Manager auf der linken Seite aus.
 
-Überprüfen Sie, ob Sie Microsoft Basic Display Adapter wie im folgenden Bild benutzen:
+Überprüfe, ob du Microsoft Basic Display Adapter wie im folgenden Bild benutzt:
 
-![Device Manager](img/DE-devicemanager.png "Device Manager")
+![Geräte-Manager](img/devicemanager-DE.png "Geräte-Manager")
 
-Bitte laden Sie sich den Treiber von der Webseite des Herstellers Ihrer Grafikkarte herunter. Hier paar allgemeine Links:
+Bitte lade dir den Treiber von der Webseite des Herstellers deiner Grafikkarte herunter. Hier ein paar allgemeine Links:
 
 - [AMD / ATI](https://amd.com/de/support)
 - [NVIDIA](https://nvidia.de/Download/index.aspx?lang=de)
 - [Intel Grafikkartentreiber](https://downloadcenter.intel.com/de/product/81500/Intel-HD-Grafik-3000)
 
 ## Falsche Bittiefe
 
-Ihr Treiber mag vielleicht richtig sein, aber die Farbbittiefe kann dennoch möglicherweise falsch sein. Windows wird dann den Ersatztreiber verwenden, [wenn die Farbbittiefe nicht 32bpp ist](https://opengl.org/discussion_boards/showthread.php/145008-Why-my-OpenGL-program-uses-Microsoft-GDI-renderer-instead-of-my-GeForce-5200). Das Ändern [der Farbbittiefe zu 32bpp sollte das Problem lösen](https://windows.microsoft.com/en-us/windows/getting-best-display-monitor#getting-best-display-monitor&section_2).
+Dein Treiber mag vielleicht richtig sein, aber die Farbbittiefe kann dennoch möglicherweise falsch sein. Windows wird dann den Ersatztreiber verwenden, [wenn die Farbbittiefe nicht 32bpp ist](https://opengl.org/discussion_boards/showthread.php/145008-Why-my-OpenGL-program-uses-Microsoft-GDI-renderer-instead-of-my-GeForce-5200). Das Ändern [der Farbbittiefe zu 32bpp sollte das Problem lösen](https://windows.microsoft.com/en-us/windows/getting-best-display-monitor#getting-best-display-monitor&section_2).
```

Comment:
```suggestion
Dein Treiber mag vielleicht richtig sein, aber die Farbbittiefe kann dennoch möglicherweise falsch sein. Windows wird dann den Ersatztreiber verwenden, [wenn die Farbbittiefe nicht 32bpp ist](https://opengl.org/discussion_boards/showthread.php/145008-Why-my-OpenGL-program-uses-Microsoft-GDI-renderer-instead-of-my-GeForce-5200). Das Ändern [der Farbbittiefe zu 32bpp sollte das Problem lösen](https://support.microsoft.com/de-de/windows/erreichen-der-besten-anzeigeeigenschaften-f%C3%BCr-ihren-bildschirm-c7e01f63-9b51-2b23-0a0f-6b965af015a9).
```

Dead link. Don't think I can properly reference the path about color depth specifically.